docs/content/tutorial/building-an-asset-graph.mdx

```diff
@@ -46,20 +52,13 @@ def topstories(topstory_ids):  # this asset is dependent on topstory_ids
             print(f"Got {len(results)} items so far.")  # noqa: T201
 
     df = pd.DataFrame(results)
-
-    return df
+    df.to_csv("data/topstories.csv")
 ```
 
-Dependencies between assets are defined by adding the asset name as a parameter to the dependent asset's function. In this case, `topstory_ids` (the list of IDs) is a dependency of `topstories` (the DataFrame).
+Dependencies between assets are defined using the `deps` parameter of the `@asset` decorator. In this case, `topstory_ids` (the list of IDs) is a dependency of `topstories` (the DataFrame).
```

Comment:
Does it still make sense to say "the DataFrame" here, given that we're not returning a DataFrame?

Reply:
yeah i suppose it's really the csv file that's the asset